### DISCUSSION TITLE
[Bug]:  Is there any way in testcontainers using which ssl for server is tested ?

### DISCUSSION THREAD

**Discussion by Shalaka1197** [testcontainers/testcontainers-java#9196] [Q&A]
### Module

PostgreSQL

### Testcontainers version

postgres:15

### Using the latest Testcontainers version?

Yes

### Host OS

Windows

### Host Arch

x86

### Docker version

```shell
4.9.0
```

### What happened?

I am using postgres test containers. I need to test ssl with postgres. is there any test cases for this ?

**Comment by eddumelendez** [comment]
Hi, you can configure postgres running in a container. I found a [gist](https://gist.github.com/mrw34/c97bb03ea1054afb551886ffc8b63c3b) about how to do it.

You can use `withEnv`, `withCopyFileToContainer` and `withCommand` in order to customize the existing `PostgreSQLContainer`
